src/model/control_desk.py

```diff
@@ -247,24 +273,25 @@ def add_bank(self, bank: FaderBank):
         Warning: This operation is expensive and might interrupt the interactions of the user. Add all columns to the
         bank at first. If all you'd like to do is updating a column: call the update function on that column.
         """
-        self.columns.append(bank)
+        self.columns.append(bank)  # TODO ein BankSet has no columns attribute
         self.update()
 
     def set_active_bank(self, i: int) -> bool:
         """This method sets the active bank.
 
         Returns:
-        True if the operation was successful. Otherwise False.
+        True if the operation was successful, Otherwise False.
         """
         if i < 0 or i >= len(self.banks):
             return False
         self.active_bank = i
         text = self.description
-        BankSet._seven_seg_data = (str(i % 100) if i > 9 else "0" + str(i)) + text[-10:] + (" " * (10-len(text)))
+        BankSet._seven_seg_data = (str(i % 100) if i > 9 else "0" + str(i)) + text[-10:] + (" " * (10 - len(text)))
         self._send_desk_update_message()
         return True
 
     def link(self) -> bool:
+        """TODO docstring"""
```

Comment:
"""Load the bank set to fish.

If there was no bank set linked before this bank set might become the active one immediately."""